packages/rolldown/src/index.ts

```diff
@@ -96,10 +95,9 @@ import type {
 } from './types/rolldown-output';
 import type { ExistingRawSourceMap, SourceMapInput } from './types/sourcemap';
 import type { PartialNull } from './types/utils';
-import { defineConfig } from './utils/define-config';
 
-export { build, defineConfig, rolldown, watch };
-export const VERSION: string = version;
+export * from './config';
```

Comment:
`loadConfig` is also exported here, which cannot run on browser build.

Maybe we should not export all, but 

```suggestion
export { defineConfig, VERSION } from './config';
```

Reply:
The initial purpose of the PR is to expose `loadConfig`, I guess doing it through `/config` is easier than doing some conditional based on if it's a browser build or not?